util/mineflayer.js

```diff
@@ -68,11 +62,7 @@ function initialize(bot) {
 		// =====
 		bot.on("breath", () => {
 			if (status.mineflayer === "true" && status.inQueue === "false") {
-				if (bot.oxygenLevel < 20) {
-					bot.setControlState("jump", true);
-				} else {
-					bot.setControlState("jump", false);
-				}
+				bot.setControlState("jump", bot.oxygenLevel < 20 ? true : false);
```

Comment:
```suggestion
				bot.setControlState("jump", bot.oxygenLevel < 20);
```
Remove unnecessary ternary.

Reply:
I wonder why eslint didn't get that one when I ran `pnpm lint`

Reply:
I wonder why eslint didn't get that one when I ran `pnpm lint`